pages/common/lpr.md

```diff
@@ -0,0 +1,31 @@
+# lpr
+
+> CUPS tool for printing files.
+
+- Print a file to the default printer:
+
+`lpr {{path/to/file}}`
+
+- Print a certain number of copies:
+
+`lpr -# {{copies}} {{path/to/file}}`
+
+- Print to a named printer:
+
+`lpr -P {{printer}} {{path/to/file}}`
+
+- Print double sided in portrait:
+
+`lpr -o sides=two-sided-long-edge {{path/to/file}}`
+
+- Print double sided in landscape:
+
+`lpr -o sides=two-sided-short-edge {{path/to/file}}`
+
+- Set page size (a4, letter, legal, etc):
+
+`lpr -o media={{paper_type}} {{path/to/file}}`
```

Comment:
Instead of paper_type, we can do a4|letter|legal. And change the description to "Set page size"

Reply:
I think there are a large number of different paper sizes, so `{{a4|letter|legal|...}}` might be a better choice.